Dockerfile

```diff
@@ -36,26 +37,31 @@ RUN true \
 FROM base as build
 LABEL maintainer="Denys Zhdanov <denis.zhdanov@gmail.com>"
 
+ARG python_binary
```

Comment:
Once this PR was introduced running project with `docker-compose` fails due to missing building argument, e.g.:

```
build:
  args:
    python_binary: python3
```

Reply:
Thanks! fixed in master